src/test/components/GlobalTrack.test.js

```diff
@@ -52,26 +53,31 @@ describe('timeline/GlobalTrack', function() {
     jest
       .spyOn(HTMLCanvasElement.prototype, 'getContext')
       .mockImplementation(() => mockCanvasContext());
+    jest
+      .spyOn(HTMLElement.prototype, 'getBoundingClientRect')
+      .mockImplementation(() => getBoundingBox(400, 400));
```

Comment:
I don't exactly know why, but without this I got some intermittents: the canvas size wasn't always the same and the snapshot wasn't matching sometimes. (same for `LocalTrack`)